session/interface.go

```diff
@@ -141,4 +141,7 @@ type Store interface {
 	// to the session with the given local pub key.
 	UpdateSessionRemotePubKey(localPubKey,
 		remotePubKey *btcec.PublicKey) error
+
+	// GetSessionByID fetches the session with the given ID.
+	GetSessionByID(id ID) (*Session, error)
```

Comment:
Since a session is only 4 bytes, there could be collisions... Do we need to return multiple sessions and then do the final comparison on the actual unique identifier which is the local key?

Reply:
addressed with latest push